asciidoc-confluence-publisher-client/src/test/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherTest.java

```diff
@@ -67,6 +67,7 @@
 
     private static final String TEST_RESOURCES = "src/test/resources/org/sahli/asciidoc/confluence/publisher/client";
     private static final String SOME_CONFLUENCE_CONTENT_SHA256_HASH = "7a901829ba6a0b6f7f084ae4313bdb5d83bc2c4ea21b452ba7073c0b0c60faae";
+    private static final String VERSIONED_MESSAGE = "versioned message";
```

Comment:
Maybe `VERSION_MESSAGE` (without `D`) would match better.

Actually, for most of the values, we prefer to use literals directly in the test, in order to see the actual value directly within the test, not just the constant name (as long as the literal communicates its meaning, see all the literal strings in the tests). Thus, you could also just inline the constant and use `version message` as a literal.